tests/test_collection.py

```diff
@@ -9,6 +9,28 @@
 from tests.helpers import assert_raises, extract, generate_col_name
 
 
+def test_get_collection_information(client):
+    # Test get information about collection
+    collection_name = generate_col_name()
+    client.create_collection(collection_name)
+    get_col_info = client.get_collection_information(collection_name=collection_name)
+    assert get_col_info['error'] is False
+    assert get_col_info['name'] == collection_name
+
+
+def test_collection_figures(client):
+    # Test get properties
+    collection_name = generate_col_name()
+    client.create_collection(collection_name)
```

Comment:
We can reuse "col" collection here but before that just make sure to useFabric "tst_fabric_name"